src/components/Nav/Nav.jsx

```diff
@@ -1,11 +1,70 @@
+import {useRef} from 'react'
+import { NavLink } from 'react-router-dom'
 import './Nav.scss'
 
 const Nav = () => {
+
+    const navRef = useRef();
+
+    const showNavbar = () => {
+        navRef.current.classList.toggle('responsive_nav')
+        // Scrolling to top of the page
```

Comment:
ten kod się sam tłumaczy, jeśli zostawiasz te komentarze dla siebie, to spoko, ale ja bym usunął